### PR TITLE
fix slack notification

### PR DESCRIPTION
**Pull request by aatoyan**

### CODE REVIEW

File: vars/folioexecuteDataMigrationUtils.groovy [hunk 1]
```diff
@@ -12,6 +12,19 @@ import org.folio.client.jira.JiraClient
 import org.folio.client.jira.model.JiraIssue
 import org.folio.karate.teams.TeamAssignment
 
+
+def getSlackColor(def buildStatus) {
+    if (buildStatus == 'STARTED') {
+        '#D4DADF'
+    } else if (buildStatus == 'SUCCESS') {
```
Comment (eldiiar-duishenaliev) [inline vars/folioexecuteDataMigrationUtils.groovy:19]: case operator would look a bit more logical in this place.

File: vars/folioexecuteDataMigrationUtils.groovy [hunk 2]
```diff
@@ -197,7 +210,7 @@ void sendSlackNotification(String slackChannel, Integer totalTimeInMs = null, Li
     }
 
     try {
-        slackSend(color: karateTestUtils.getSlackColor(buildStatus), message: message, channel: slackChannel)
```
Comment (eldiiar-duishenaliev) [inline vars/folioexecuteDataMigrationUtils.groovy:200]: are you sure about this change?

**Review by eldiiar-duishenaliev — CHANGES_REQUESTED** [review]
Please review comments.

File: vars/folioexecuteDataMigrationUtils.groovy (hunk 2 above)
Comment (aatoyan) [reply vars/folioexecuteDataMigrationUtils.groovy:200]: @eldiiar-duishenaliev it was tested and it works, what is your concern?

File: vars/folioexecuteDataMigrationUtils.groovy (hunk 1 above)
Comment (OHaimanov) [reply vars/folioexecuteDataMigrationUtils.groovy:19]: Agree, switch case better to use here
Or Map

Comment (aatoyan) [reply vars/folioexecuteDataMigrationUtils.groovy]: My dear friends kindly ask you to review it again

**Review by yaroslavishchenko — APPROVED** [review]

**Review by eldiiar-duishenaliev — APPROVED** [review]

**Pull request merged**